modules/tinymce/src/themes/silver/main/ts/ui/dialog/Tree.ts

```diff
@@ -9,6 +11,10 @@ import * as Icons from '../icons/Icons';
 
 type TreeSpec = Omit<Dialog.Tree, 'type'>;
 type OnLeafAction = (id: string) => void;
+interface ExpandTreeNodeEventArgs extends EventFormat {
```

Comment:
Maybe call it ToggleExpand since it's both expand and collapse at the same time?

Also add a blank line before the interface. The other interfaces in this file all have blank lines between them. It kind of a block of code. I think the type declarations are fine to have next to each other. I think we should have a lint rule for this. Maybe it's just me nit picking. :smile: 

```suggestion

interface ToggleExpandTreeNodeEventArgs extends EventFormat {
```

Reply:
Done!